packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -36,6 +36,28 @@ const Color _kLightThemeSplashColor = const Color(0x66C8C8C8);
 const Color _kDarkThemeHighlightColor = const Color(0x40CCCCCC);
 const Color _kDarkThemeSplashColor = const Color(0x40CCCCCC);
 
+/// Configures the tap target and layout size of certain Material widgets.
+///
+/// The following widgets are impacted:
+///
+///   * [FloatingActionButton], only the mini tap target size is increased.
+///   * [MaterialButton]
+///   * [OutlineButton]
+///   * [FlatButton]
+///   * [TimePicker]
+///   * [SnackBar]
+enum MaterialTapTargetSize {
+  /// Expands the minimum tap target size to 48dp by 48dp.
+  ///
+  /// This is the default value of [ThemeData.materialHitTestSize] and the
+  /// recommended behavior to conform to Android accessibility recommendations.
+  expanded,
+
+  /// Collapses the tap target size to the minimum provided by the Material
+  /// specification.
```

Comment:
cc @goderbauer for wording on this, but I'm guessing we should just say something like "Tap targets will exactly fit around the drawn pixels of the widget", to avoid making any claims and the spec.

Reply:
Android accessibility scanner does explicitly recommend 48 by 48, agree phrasing could be better

Reply:
Referring to the recommendations from Android Accessibility Scanner seems fine here and maybe even desirable to warn people that changing this value will affect the accessibility experience.